packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -71,6 +72,7 @@ export const AuthContext = React.createContext({} as Auth)
                     siteId="RefArch"
                     shortCode="12345678"
                     locale="en-US"
+                    enablePWAKitPrivateClient={true}
```

Comment:
Can we add some more unit for this change please?

Reply:
Could you please clarify what kind of unit test you would like to see?

All the private client tests in commerce-sdk-react's `auth/index.test.tx` use a configuration `configSLASPrivate` where `enablePWAKitPrivateClient` is true. Other tests where private client is not needed use a different config where the only difference is that `enablePWAKitPrivateClient` is not set to true.

Reply:
We talked about this, and all the test cases are covered in auth.test.ts